internal/cluster/validator.go

```diff
@@ -294,18 +294,28 @@ func (v *clusterValidator) printSufficientMastersCount(context *clusterPreproces
 }
 
 func (v *clusterValidator) allHostsAreReadyToInstall(c *clusterPreprocessContext) validationStatus {
+	numberOfEnabledHosts := 0
 	foundNotKnownHost := false
 	for _, host := range c.cluster.Hosts {
-		if swag.StringValue(host.Status) != models.HostStatusDisabled && swag.StringValue(host.Status) != models.HostStatusKnown {
-			foundNotKnownHost = true
-			break
+		if swag.StringValue(host.Status) != models.HostStatusDisabled {
+			numberOfEnabledHosts++
+			if swag.StringValue(host.Status) != models.HostStatusKnown {
+				foundNotKnownHost = true
+			}
+
 		}
 	}
+	if numberOfEnabledHosts < common.MinMasterHostsNeededForInstallation {
```

Comment:
Usually, we use pending validation to indicate that this validation depends on other validation.  This is not the case here. The UI hides pending validations.  
So the pending status should be removed.
The same information is received by "sufficient master count" and "all hosts ready to install"

Reply:
@ori-amizur as you can see in the Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1872668 @avishayt  agreed that it should be pending.

Reply:
So this validation will be hidden by UI in this case.  I think it is wrong.
A better solution will be to display a different validation message in this case and the result should be failure.

Reply:
numberOfEnabledHosts contains also workers so it can not be compared to MinMasterHostNeededForInstallation

Reply:
there was a bug that if we have only 2 known hosts in the cluster - the validation of "All hosts are ready to install" return true instead of false - the role of the host is irrelevant if we have only 1 or 2 hosts that are registered and in known state